src/app/views/home/home.component.scss

```diff
@@ -40,47 +48,74 @@
       color: $color-primary-cerise;
     }
 
-    @media screen and (min-width: 576px) {
+    @media (min-width: 576px) {
```

Comment:
This value is coupled to another, so if one changes the other needs to be changed too.

__suggestion__

```sass
$any-variable-name-upper: 576px;
$any-variable-name: $any-variable-name-upper -1px;
```